src/components/ProteinViewer/RepresentativeDomainsTrack/index.tsx

```diff
@@ -90,14 +90,9 @@ const RepresentativeDomainsTrack = ({
           use-ctrl-to-zoom
         />
       </div>
-      {entries.length === 1 ? (
-        <LabelsInTrack
-          entry={entries[0]}
-          hideCategory={hideCategory}
-          expandedTrack={true}
-          isPrinting={isPrinting}
-        />
-      ) : null}
+      <div className={css('track-label', 'centered-label')}>
+        <b>Representative</b>
```

Comment:
Ideally, we could pass the label as a props of `RepresentativeDomainsTrack` so the representative domains track is labelled "Representative domains", and similarly for representative families.